.circleci/config.yml

```diff
@@ -44,14 +44,15 @@ jobs:
     steps:
       - add_ssh_keys
       - checkout
-      - attach_workspace:
-          at: /tmp/workspace
```

Comment:
This wasn't about heroku, this was a folder on circleci you can put files in on one job and then use them in another. Lets you pass things through the workflow. Before the manifest file was created in the heroku deploy and then used again in the hokusai deploy but now we just generate it as part of the hokusai deploy since there isn't a heroku one.

Reply:
Ohhh looks like I was totally lost there haha. Thanks for explaining! 👍